src/docfx/Models/BuildCommandOptions.cs

```diff
@@ -123,7 +123,10 @@ internal class BuildCommandOptions : ICanPrintHelpMessage, ILoggable
         [OptionList("postProcessors", Separator = ',', HelpText = "Set the order of post processors in plugins")]
         public List<string> PostProcessors { get; set; }
 
-        [OptionList("lruSize", HelpText = "Set the LRU cached model count (approximately the same as the count of input files). By default, it is 8192 for 64bit and 3072 for 32bit process. With LRU cache enabled, memory usage decreases and time consumed increases. If set to 0, Lru cache is disabled.")]
+        [Option("lruSize", HelpText = "Set the LRU cached model count (approximately the same as the count of input files). By default, it is 8192 for 64bit and 3072 for 32bit process. With LRU cache enabled, memory usage decreases and time consumed increases. If set to 0, Lru cache is disabled.")]
         public int? LruSize { get; set; }
+
+        [Option("noDereference", HelpText = "If set to true, manifest will keep links form.")]
```

Comment:
>links form [](start = 80, length = 10)

minor: add explanation for what's "links from"? how about "keep original links"?